tests/functional.rs

```diff
@@ -638,8 +638,15 @@ async fn functional() {
     info("##################################################################");
     info("");
     {
-        delete_ca(&ca4).await;
+        // Remove CA4 from CA3
+        delete_parent(&ca4, &ca3).await;
+
+        // Expect that CA4 withdraws all
+        {
+            assert!(will_publish("CA4 should withdraw objects when parent is removed", &ca4, &[]).await);
```

Comment:
Stupid question: where is it tested that the objects are withdrawn?

Reply:
No, you're right.. added another test for this - will now delete CA3 and expect it to withdraw everything, and its parent to revoke and withdraw the .cer issued to it.